src/pe/header.rs

```diff
@@ -587,6 +587,145 @@ impl ctx::TryIntoCtx<scroll::Endian> for Header {
     }
 }
 
+/// The TE header is a reduced PE32/PE32+ header containing only fields required for
+/// execution in the PI architecture. The TE header is described in by the PI spec:
```

Comment:
nit: I'm not familiar with PI architecture, might be good to mention what this acronym is/stands for elsewhere or before use in docs?

Reply:
Oh yes. That makes sense! I've updated the PR comment, and a few areas in the documentation to specify the acronym (Platform Initialization) and added a link to the specification.